_docs/versions.md

```diff
@@ -28,4 +28,19 @@ Let's detail some of the versions being specified:
   This is only supported by Portus in the `master` branch and in `2.0.3`.
 - Docker Distribution `2.3` supports both Manifest versions, but some changes
   had to be made in order to offer backwards compatibility. This is not
-  supported neither for Portus `2.0.0` nor `2.0.1`.
+  supported neither for Portus `2.0.0` nor `2.0.1`. Moreover, Docker
+  Distribution `2.4` was the first to introduce garbage collection. Because of
+  this we recommend running at least `2.4` if you are using `2.1` (or higher),
```

Comment:
To make the versions a bit more clear, I'd suggest writing something similar to this:

    Because of this, we recommend running Docker Distribution version `2.4` (or higher) if you are using Docker version `2.1` or later.